work-locally.mdx

```diff
@@ -10,21 +10,20 @@ experimentation with new AI models.
 
 ## Hyp CLI setup
 
-For access to hosted models, install the Hyp CLI to augment Modus.
+1. First, install the Hyp CLI to augment Modus:
```

Comment:
I don't particularly want to use this everywhere, but instead of numbers should we use Mintlify's steps? https://mintlify.com/docs/content/components/steps

Reply:
Ooh, yes, I think that would work nicely. I'll try it out.